mcs/class/corlib/corert/Decimal.cs

```diff
@@ -1219,13 +1214,20 @@ internal int Scale
 
         private ulong Low64
         {
-#if BIGENDIAN && !MONO
-            get { return ((ulong)umid << 32) | ulo; }
-            set { umid = (uint)(value >> 32); ulo = (uint)value; }
-#else
-            get => ulomidLE;
-            set => ulomidLE = value;
-#endif
+            get
+            {
+                if (BitConverter.IsLittleEndian)
+                    return ulomidLE;
+                else
+                    return ((ulong)umid << 32) | ulo;
+            }
+            set
+            {
+               if (BitConverter.IsLittleEndian)
+                   ulomidLE = value;
+               else
+                   umid = (uint)(value >> 32); ulo = (uint)value;
```

Comment:
Isn't this supposed to be wrapped in { } ?

Reply:
heh 🙂